### PR TITLE
sig-testing meets on tuesdays

### PR DESCRIPTION
**Pull request by spiffxp**
whoops, missed this in all the commotion with #3

### CODE REVIEW

File: sig-testing/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The Kubernetes Testing SIG (sig-testing) is a working group within the Kubernete
 
 ## video conference
 
-We meet weekly at 9:30am PDT (16:30pm UTC) at [this zoom room](https://zoom.us/j/553910341)
```
Comment (ixdy) [inline sig-testing/README.md]: we should probably say either 16:30 or 4:30pm, not both :)

Comment (spiffxp) [inline sig-testing/README.md]: bahaha good catch

**Comment by spiffxp** [comment]
self merging after waiting, per CONTRIBUTING.md

**Pull request merged**